spec/Condition/QueryEvaluatorTwofoldSpec.php

```diff
@@ -0,0 +1,94 @@
+<?php
+namespace spec\Riesenia\Utility\Condition;
+
+use PhpSpec\ObjectBehavior;
+use Riesenia\Utility\Condition\QueryEvaluatorException;
+
+class QueryEvaluatorTwofoldSpec extends ObjectBehavior
+{
+    public function let()
+    {
+        $this->beConstructedWith([
+            'P1' => [
+                'pid' => [
+                    'field' => 'id',
+                    'operators' => ['=', 'NOT', 'IN', 'NOTIN']
+                ],
+                'name' => [
+                    'field' => 'name',
+                    'operators' => ['=', 'NOT', 'CONTAINS']
+                ],
+                'price' => [
+                    'field' => 'unit_price',
+                    'operators' => ['>=', '>', '<', '<='] ]
+                ],
+            'P2' => [
+                'pid' => [
+                    'field' => 'id',
+                    'operators' => ['=', 'NOT', 'IN', 'NOTIN']
+                ],
+                'name' => [
+                    'field' => 'name',
+                    'operators' => ['=', 'NOT', 'CONTAINS']
+                ],
+                'price' => [
+                    'field' => 'unit_price',
+                    'operators' => ['>=', '>', '<', '<=']
+                ]
+            ]
+        ]);
+    }
+
+    public function it_parses_simple_conditions()
+    {
+        $this->parse('P1.pid = P2.pid')->shouldReturn(['P1.id  P2.id']);
```

Comment:
this should return `P1.id = P2.id`

Reply:
yes, that's what i assumed at first but when i tested it in the actual code that's what it gives me.
<img width="480" alt="2017-09-15_0635" src="https://user-images.githubusercontent.com/253609/30458895-3ed600b4-99e0-11e7-9ab4-8ac05ef1a34d.png">

and in QueryEvaluator() class, _parseCondition() method, the when it receives a "="
it returns it back as empty string.
<img width="658" alt="2017-09-15_0637" src="https://user-images.githubusercontent.com/253609/30458970-87dd3854-99e0-11e7-94f0-f6c1e66d2ee4.png">

am i missing something here?

Reply:
fixed

Reply:
i am not sure if this was clear. the code should be:
`$this->parse('P1.pid = P2.pid')->shouldReturn(['P1.id = P2.id']);`